sdk/python/tests/integration/online_store/test_universal_online.py

```diff
@@ -715,7 +715,7 @@ def eventually_apply() -> Tuple[None, bool]:
     assert all(v is None for v in online_features["value"])
 
 
-@pytest.mark.skip
+#@pytest.mark.skip
```

Comment:
prefer to remove the line rather than comment

Reply:
Oops this was an accident.